docs/content/docs/deployment/config.md

```diff
@@ -289,6 +289,15 @@ Enabling RocksDB's native metrics may cause degraded performance and should be s
 ----
 ----
 
+# Traces
```

Comment:
NIT: This doesn't show up in TOC - it should be 2 or more `#` with the current settings.

As well as other, existing section (e.g. `History Server` is missing).

Settings can be fixed by adding to `docs/config.toml`:
```
[markup.tableOfContents]
startLevel = 1
```
Not sure if this change would be reasonable and whether it can be added as a hotfix to this PR.

Reply:
Huh, good catch. I was curious a couple of times why the TOC seem broken, but never investigated it. I think this is a good fix, but let's ask @zentol  maybe.

Reply:
I think this is intentional such that the main heading isn't part of the ToC.

> we shouldn't have more than a single Header lvl 1

correct, this is probably also why the Configuration ToC is so crap.

Reply:
I've clicked through quite a lot of pages, and I didn't find anything too concerning or broken. One can argue that the extra level of indentation on most pages slightly reduces readability, as ToC is smaller or takes more space, but that's in an exchange of showing the title in the ToC. I think that's a fair trade.

And in those pages that have multiple headers lvl 1 ToC obviously is much much better.